config/default.yaml

```diff
@@ -20,15 +20,19 @@
 vm_name    : 'dspace-dev'
 
 # How much memory to provide to VirtualBox VM (in MB)
-# Provide 2GB of memory by default
+# Provide 4GB of memory by default
 # (Changes to this setting require a 'vagrant reload' to take effect)
-vm_memory  : 2048
+vm_memory  : 4096
```

Comment:
Just as a note, I run vagrant-dspace with this set to 3072 (3GB) with no issues.  Requiring 4GB RAM by default seems slightly excessive if we want vagrant-dspace to also appeal to folks who simply want to "try out" DSpace before you install.  I'd recommend setting this to the "lowest common denominator" (i.e. the lowest reasonable level where vagrant-dspace still functions).  Individual developers can easily bump this up if they have more memory to work with.